gateway/src/main/resources/application.yaml

```
@@ -0,0 +1,33 @@
+spring:
+  application:
+    name: gateway
+  cloud:
+    gateway:
+      routes:
+        - id: driver
+          uri: http://localhost:8081
+          predicates:
+            - Path=${commonApiPath}/cars/**, ${commonApiPath}/drivers/**
```

Не следует жёстко задавать uri таким способом, ведь 1) он может поменяться в зависимости от окружения, 2) в приложение подключена Eureka, следует её использовать. Перепроверь такой вариант:
```suggestion
  cloud:
    gateway:
      routes:
        - id: driver
          uri: lb://driver-service // здесь должно быть имя приложения, как оно зарегистрировано в Eureka
          predicates:
            - Path=${commonApiPath}/cars/**, ${commonApiPath}/drivers/**
```